lib/bios/seabios.robot

```diff
@@ -0,0 +1,305 @@
+*** Settings ***
+Documentation       Collection of keywords related to SeaBIOS.
+
+# Here is comparison of terms with lib/bios/menus.robot
```

Comment:
I can do this, but it would require a force push, which will close all your comments. Maybe this should be the last step. Do you have a better idea of how to address this comment?

Reply:
We can also ignore it, I have already diffed manually

Reply:
Shall we close this?